external-api/api-tests/v1/bank-holidays.api-test.ts

```diff
@@ -0,0 +1,70 @@
+import { app } from '../../src/createApp';
+import { api } from '../api';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import { mockResponseBankHolidays } from '../test-mocks/bank-holidays';
+import { getYear, addYears } from 'date-fns';
+import { BankHolidaysEvent } from '../../src/interfaces';
+
+const { get } = api(app);
+
+describe('/bank-holidays', () => {
+  const mock = new MockAdapter(axios);
+  jest.mock('axios', () => jest.requireActual('axios'));
```

Comment:
I can see this has been used elsewhere, but is it needed? I may not be understanding this correctly, but this seems to be saying "mock axios and replace it with axios"?